### PR TITLE
Package globlon.1.0

### PR DESCRIPTION
**Pull request by bleepbloopsify**
### `globlon.1.0`
A globbing library for OCaml
A globbing library for OCaml. We use it to match file paths against patterns.

---
* Homepage: https://github.com/bleepbloopsify/globlon
* Source repo: git+https://github.com/bleepbloopsify/globlon.git
* Bug tracker: https://github.com/bleepbloopsify/globlon/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/globlon/globlon.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A globbing library for OCaml"
+description:
+  "A globbing library for OCaml. We use it to match file paths against patterns."
+maintainer: ["bleepbloopisfy"]
+authors: ["bleepbloopisfy"]
+license: "MIT"
+homepage: "https://github.com/bleepbloopisfy/globlon"
+bug-reports: "https://github.com/bleepbloopisfy/globlon/issues"
```
Comment (kit-ty-kate) [inline packages/globlon/globlon.1.0/opam:10]:
```suggestion
homepage: "https://github.com/bleepbloopsify/globlon"
bug-reports: "https://github.com/bleepbloopsify/globlon/issues"
```

File: packages/globlon/globlon.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/bleepbloopisfy/globlon.git"
```
Comment (kit-ty-kate) [inline packages/globlon/globlon.1.0/opam:29]:
```suggestion
dev-repo: "git+https://github.com/bleepbloopsify/globlon.git"
```

File: packages/globlon/globlon.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A globbing library for OCaml"
+description:
+  "A globbing library for OCaml. We use it to match file paths against patterns."
+maintainer: ["bleepbloopisfy"]
+authors: ["bleepbloopisfy"]
```
Comment (kit-ty-kate) [inline packages/globlon/globlon.1.0/opam:7]: any chance you could add an email address for people who don’t use github?

**Review by kit-ty-kate — APPROVED** [review]
Looks good overall apart from some minor comments

File: packages/globlon/globlon.1.0/opam (hunk 3 above)
Comment (bleepbloopsify) [reply packages/globlon/globlon.1.0/opam]: done, added the email address and left the existing username

File: packages/globlon/globlon.1.0/opam (hunk 2 above)
Comment (bleepbloopsify) [reply packages/globlon/globlon.1.0/opam]: done, updated in my dune-project, and ran `opam publish` again

File: packages/globlon/globlon.1.0/opam (hunk 1 above)
Comment (bleepbloopsify) [reply packages/globlon/globlon.1.0/opam]: done (I can't spell my own name? lol)

**Comment by dinosaure** [comment]
Thanks!

**Pull request merged**